scripts/o3de/o3de/engine_template.py

```diff
@@ -257,8 +257,8 @@ def _execute_template_json(json_data: dict,
     for copy_file in json_data['copyFiles']:
         # construct the input file name
         in_file = template_path / 'Template' / copy_file['file']
-
-        # construct the output file name
+        
+        # construct the output file name        
```

Comment:
There is extra whitespace here
```suggestion

        # construct the output file name
```

Reply:
Thank you!